apps/src/p5lab/spritelab/commands/criterionCommands.js

```diff
@@ -46,30 +46,31 @@ export const commands = {
 
   // Return true if any sprite began speaking
```

Comment:
nit: update comment for this and the methods below to reference the min variable